config/initializers/0_clowder-config.rb

```diff
@@ -19,13 +19,25 @@
   redis_url = "#{config.dig('inMemoryDb', 'hostname')}:#{config.dig('inMemoryDb', 'port')}"
   redis_password = config.dig('inMemoryDb', 'password')
 
+  # Kafka
+  first_kafka_server_config = config.kafka.brokers[0]
+
+  kafka_server_config = {
+    brokers: "#{first_kafka_server_config.dig('hostname')}:#{first_kafka_server_config.dig('port')}",
```

Comment:
(Continuation of https://github.com/RedHatInsights/compliance-backend/pull/1025#discussion_r743691416)

Can we put all brokers locations here? We can keep reading the base configuration (auth, cacert) from the first one.

Reply:
yeah I thought that too - however I don't know how that will work , if ... there's actual multiple brokers, each with different auth. modes

Reply:
I've re-added all the broker locations as suggested and used the first broker's config for setting up the authentication config.